js/app-super-resolution/README.md

```diff
@@ -0,0 +1,59 @@
+# RUNNING SUPER-RESOLUTION ON IMAGES ACROSS VARIOUS PLATFORMS USING ONNX RUNTIME MODELS
+## Examples
+1. MOBILE
+  <img width="318" alt="image" src="https://user-images.githubusercontent.com/106185642/181639530-9c808167-d68c-49d4-8e89-72aeeb11164e.png">
+
+2. WEB
+  <img width="1215" alt="image" src="https://user-images.githubusercontent.com/106185642/181638855-f341e52e-dfc1-4362-b93a-0117f0cfd65a.png">
+   
+
+## Steps of building project from scratch
+0. Prerequisites
+    1. Install Node.js
+    2. Install Java and OpenJDK
+    3. Install expo
+       ```sh
+       npm install -g expo-cli
+       ```
+    4. Install yarn
+       ```sh
+       npm install -g yarn
+       ```
+       
+1. Setup project
+    1. Create Root Directory
+    2. Copy contents of this Repo into created directory
+
+    **NOTE:**
+    - `<SOURCE_ROOT>` refers to the root folder of the source code directory.
```

Comment:
Is this the root of the onnxruntime-inference-examples, or the app directory?

Do they need to copy, or can the clone the examples repo and change directories into the app folder?

Reply:
It refers to the app directory. They can clone the examples repo and change directories into the app folder. I will specify that in the README